src/pages/ProfilePage/HelpCenter.tsx

```
@@ -0,0 +1,140 @@
+        Help Center
+      </h2>
+
+      <div className="mb-6 flex rounded-full border border-neutral-20 p-1">
+        <button
+          onClick={() => setActiveTab("faq")}
+          className={`flex-1 rounded-full py-2 text-sm font-medium transition ${
+            activeTab === "faq"
+              ? "bg-primary text-white"
+              : "text-muted-foreground"
+          }`}
+        >
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)